public/app/core/components/NavBar/utils.test.ts

```diff
@@ -193,6 +193,13 @@ describe('getActiveItem', () => {
       url: '/d/moreSpecificDashboard',
     },
   ];
+  beforeEach(() => {
+    locationUtil.initialize({
+      config: { appSubUrl: '/subUrl' } as any,
+      getVariablesUrlParams: (() => {}) as any,
+      getTimeRangeForUrl: (() => {}) as any,
```

Comment:
I know you probably copy-pasted this from elsewhere, but it would be appreciated it you could just update this to 

```
      getVariablesUrlParams: () => ({}),
      getTimeRangeForUrl: () => ({ from: 'now-7d', to: 'now' }),
```

Reply:
Yeah that's exactly what I did since it was written like this everywhere in the code. Still, it seems that [betterer doesn't like the `any` on config parameter](https://drone.grafana.net/grafana/grafana/69982/1/7) and I don't understand why. Also, I cant figure out why the new test case is failing...